internal/config/config.go

```diff
@@ -0,0 +1,45 @@
+package config
+
+import (
+	"bytes"
+	"fmt"
+	"text/tabwriter"
+
+	"github.com/onflow/flow-cli/pkg/flowcli/project"
+	"github.com/onflow/flow-cli/pkg/flowcli/util"
+)
+
+// todo implement top level command config
+
+// InitResult result structure
+type InitResult struct {
```

Comment:
If this struct is only used by the `init` command, maybe just move it to `init.go`

Reply:
Yes, you are right, I wanted to leave it there to start the notion of having config as top level command one day but maybe is better to not do that until there is more code.